src/main/resources/db/migration/V1_1__predictor_scores.sql

```diff
@@ -0,0 +1,18 @@
+CREATE TABLE IF NOT EXISTS offender_predictor
+(
+    offender_predictor_id           serial          primary key,
+    offender_predictor_uuid         uuid            not null unique,
+    predictor_type                  varchar(100)    not null,
+    algorithm_version               varchar(10)     not null,
+    predictor_score                 varchar(10)     not null,
```

Comment:
This is a string because it could store both numbers or text (e.g. RSR value might be 0.1111111 but RSR level might be "High"

Reply:
We will send both back in the response, maybe we should store both the scoring and the level.

`RiskPredictorsDto(rsrScore = Score(PredictorType.RSR, ScoreLevel.HIGH, BigDecimal("11.34")))`